EIPS/eip-overlay_tree.md

```diff
@@ -0,0 +1,108 @@
+---
+eip: <to be assigned>
+title: Trie format transition with overlay trees
+author: Guillaume Ballet (@gballet)
+discussions-to: https://ethresear.ch/t/overlay-method-for-hex-bin-tree-conversion/7104
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-04-03
+---
+
+## Simple Summary
+
+This EIP proposes a method to convert the state trie format from hexary to binary: new values are directly stored in a binary trie “laid over” the hexary trie. Meanwhile, the hexary trie is converted to a binary trie in the background. When the process is finished, both layers are merged.
```

Comment:
That is voluntarily left vague at the moment, because I don't know at this time if both trees are going to be merged (it has been discussed at the 1x workshop and there's an EIP by Vitalik that I haven't found yet).

I currently assume that it will be both the account and storage tries, but the exact shape of things depend on how and when other teams work this out.